src/index.html

```diff
@@ -2,11 +2,370 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logotype">
+      </a>
+      <nav class="nav">
+        <li class="nav__item">
+          <a href="#apple" class="nav__link is-active">Apple</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#samsung" class="nav__link">Samsung</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#smartphones" class="nav__link">Smartphones</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#laptops&computers" class="nav__link"
+          data-qa="nav-hover">Laptops & computers</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#gadgets" class="nav__link">Gadgets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#tablets" class="nav__link">Tablets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#photo" class="nav__link">Photo</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#video" class="nav__link">Video</a>
+        </li>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div class="card"
+        data-qa="card">
+          <img
```

Comment:
<img width="1332" alt="Screenshot 2023-06-28 at 10 20 15" src="https://github.com/mate-academy/layout_catalog/assets/53228259/9ef50745-cd57-43c0-a4d7-a3ab2ccae8c3">

Reply:
not solved